lib/vfscore/Makefile.uk

```diff
@@ -17,6 +17,14 @@ LIBVFSCORE_SRCS-y += $(LIBVFSCORE_BASE)/lookup.c
 LIBVFSCORE_SRCS-y += $(LIBVFSCORE_BASE)/fops.c
 LIBVFSCORE_SRCS-y += $(LIBVFSCORE_BASE)/subr_uio.c
 LIBVFSCORE_SRCS-y += $(LIBVFSCORE_BASE)/extra.ld
+
+ifeq ($(CONFIG_LIBUKVMEM),y)
+LIBVFSCORE_SRCS-y += $(LIBVFSCORE_BASE)/vma_file.c|isr
+ifneq ($(filter y,$(CONFIG_LIBUKVMEM_TEST) $(CONFIG_LIBUKTEST_ALL)),)
+LIBVFSCORE_SRCS-y += $(LIBVFSCORE_BASE)/tests/test_vma.c
```

Comment:
Should we maybe include this to the build conditioned by a vfscore test config in addition to it being enabled only in ukvmem is enabled? Something like
```
LIBVFSCORE_SRCS-$(LIBVFSCORE_TEST) += $(LIBVFSCORE_BASE)/tests/test_vma.c
```

Reply:
Actually, I take that back. Maybe it is better to just do something like:
LIBVFSCORE_SRCS-$(LIBVFSCORE_TEST) += $(LIBVFSCORE_BASE)/tests/test_vfscore.c
and inside that file have the addition `#if vmem` guarded. My rationale is that this would be where vfscore would have its tests and at the moment there is only the test for file VMA's. That being said, what I am thinking is that vma_file.c would become the only file in the makefile confitioned by ukvmem.